firestore/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java

```diff
@@ -328,4 +335,51 @@ void createMultipleCursorConditionsQuery() {
         .startAt("Springfield", "Missouri");
     // [END fs_multiple_cursor_conditions]
   }
+
+  /**
+   * Create a query using a snapshot as a start point.
+   *
+   * @return query
+   */
+  Query createStartAtSnapshotQueryCursor()
+      throws InterruptedException, ExecutionException, TimeoutException {
+    // [START fs_document_snapshot_cursor]
+    // Fetch the Snapshot
+    ApiFuture<DocumentSnapshot> future = db.collection("cities").document("SF").get();
+    DocumentSnapshot snapshot = future.get(30, TimeUnit.SECONDS);
```

Comment:
What does the magic number represent other than 30 seconds?

Reply:
Could you use variable names instead of a magic number to state the same information?

Reply:
Thanks, that helps!